setup.py

```diff
@@ -50,8 +50,8 @@
             'numpy>=1.7',
             'pandas>=0.13.1',
             'datacache>=0.4.2',
-	    'pyfaidx',
-	    'memoized-property',
+            'pyfaidx>=0.3.4',
+            'memoized-property>=1.0.2',
```

Comment:
Yeah, especially in a library that depends so much on external resources,
it might be nice to trouble shoot a local problem by seeing what unit tests
fail.
On Feb 23, 2015 12:10 PM, "Tavi Nathanson" <notifications@github.com> wrote:

> In setup.py
> <https://github.com/hammerlab/pyensembl/pull/44#discussion_r25181288>:
>
> > @@ -50,8 +50,8 @@
> >              'numpy>=1.7',
> >              'pandas>=0.13.1',
> >              'datacache>=0.4.2',
> > -	    'pyfaidx',
> > -	    'memoized-property',
> > +            'pyfaidx>=0.3.4',
> > +            'memoized-property>=1.0.2',
>
> Not currently: do you think they should?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/pyensembl/pull/44/files#r25181288>.
>